### PR TITLE
Replace images with the new ones

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

File: modules/ROOT/pages/data-modeling/guide-data-modeling.adoc [hunk 1]
```diff
@@ -75,15 +76,21 @@ Consider the following scenario describing our example data entities and connect
 Two _people_, *Sally* and *John*, +++<u>are friends</u>+++.
 Both *John* and *Sally* +++<u>have read</u>+++ the _book_, *Graph Databases*.
 
-
 We can use the information in this statement to build our model by identifying the components as labels, nodes, and relationships.
 Let us take the scenario into pieces and define them as parts of our property graph model.
+However, for a start, we simplify the model.
 
-ifndef::env-guide[]
 .Review - Property Graph Elements (click to zoom)
```
[the remark this answers is not in the document]
Comment (lidiazuin) [reply modules/ROOT/pages/data-modeling/guide-data-modeling.adoc:83]: I also think it's better not to capitalize every word

**Review by lidiazuin — COMMENTED** [review]
A few suggestions that could improve the images! Let me know if you need help with that :)

File: modules/ROOT/pages/data-modeling/guide-data-modeling.adoc [hunk 2]
```diff
@@ -25,46 +27,45 @@ A Neo4j graph data model is designed to answer questions in the form of Cypher q
...
-This is where the graph data model gets its name for being "whiteboard-friendly".
+This is where the graph data model gets its name for being _whiteboard-friendly_.
 
 Let us look at an example to demonstrate this.
-In the whiteboard drawing below, we have a data set about the movie "The Matrix".
+In the whiteboard drawing below, we have a data set about the movie _The Matrix_.
 
 .Matrix - whiteboard model
-image:matrix_whiteboard_model1.png[role="popup-link"]
+image::matrix_whiteboard_model1.png[600,600,role="popup-link"]
```
Comment (lidiazuin) [inline modules/ROOT/pages/data-modeling/guide-data-modeling.adoc:45]: We should aim to use only .svg files for images.

File: modules/ROOT/pages/data-modeling/guide-data-modeling.adoc [hunk 3]
```diff
@@ -75,15 +76,21 @@ Consider the following scenario describing our example data entities and connect
...
 
-
 We can use the information in this statement to build our model by identifying the components as labels, nodes, and relationships.
 Let us take the scenario into pieces and define them as parts of our property graph model.
+However, for a start, we simplify the model.
 
-ifndef::env-guide[]
 .Review - Property Graph Elements (click to zoom)
-image:property_graph_elements.jpg[role="popup-link"]
+image::property_graph_elements-arr.svg[role="popup-link"]
```
[the remark this answers is not in the document]
Comment (NataliaIvakina) [reply modules/ROOT/pages/data-modeling/guide-data-modeling.adoc]: I updated this image and the other ones to make them look more balanced on a page

[the remark this answers is not in the document]
Comment (lidiazuin) [reply modules/ROOT/pages/data-modeling/guide-data-modeling.adoc]: I think it's looking much better now!

**Pull request merged**